mw-webapp/src/component/tag/Tag.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "src/styles/variables";
 
-$WayTagPadding: 10px;
-$WayTagBorderRadius: 1000px;
+$TagPadding: 10px;
```

Comment:
let's use global var - paddingMedium with the same value

Reply:
Done